From 1ee06c3678a85d158eb82d4af438d1e43a4c814e Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 6 Dec 2020 16:14:17 +0000
Subject: [PATCH] GCC: change make_visitor visibility to public

GCC complains that make_visitor is used in private context from
inner Iterator class.
---
 net/third_party/quiche/src/quic/core/quic_interval_set.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/net/third_party/quiche/src/quic/core/quic_interval_set.h b/net/third_party/quiche/src/quic/core/quic_interval_set.h
index af64e29..7ee8978 100644
--- a/net/third_party/quiche/src/quic/core/quic_interval_set.h
+++ b/net/third_party/quiche/src/quic/core/quic_interval_set.h
@@ -1874,7 +1874,6 @@ class QUIC_NO_EXPORT QuicIntervalSet {
     return absl::visit([&](auto& s) { return s.Contains(min, max); }, qiset_);
   }
 
- private:
   template <class A, class B, class C>
   struct overloader : A, B, C {
     overloader(A a, B b, C c) : A(a), B(b), C(c) {}
-- 
2.26.2

